src/main/kotlin/com/grepp/nbe1_3_team04/team/repository/CustomTeamMemberRepositoryImpl.kt

```diff
@@ -0,0 +1,24 @@
+package com.grepp.nbe1_3_team04.team.repository
+
+import com.grepp.nbe1_3_team04.global.domain.IsDeleted
+import com.grepp.nbe1_3_team04.team.domain.QTeamMember.teamMember
+import com.grepp.nbe1_3_team04.team.domain.TeamMember
+import com.querydsl.jpa.impl.JPAQueryFactory
+
+
+class CustomTeamMemberRepositoryImpl(
+    private val queryFactory: JPAQueryFactory
+) : CustomTeamMemberRepository {
+
+    override fun findByTeamIdAndMemberId(teamId: Long, memberId: Long): TeamMember? {
+        return queryFactory
+                .select(teamMember)
+                .from(teamMember)
```

Comment:
```suggestion
                .selectFrom(teamMember)
```
select(teamMember)와 from(teamMember)가 함께 사용될 때는 이런 식으로도 작성할 수 있습니다! 고칠 필요는 없어 보이고 참고만 해 주세요😊

Reply:
오 처음 알았습니다!! 훨씬 깔끔해보이네요! 적용해 놓겠습니다
감사합니다